app/views/schemes/index.html.erb

```diff
@@ -6,7 +6,9 @@
 <%= render partial: "organisations/headings", locals: current_user.support? ? { main: "Supported housing schemes", sub: nil } : { main: "Supported housing schemes", sub: current_user.organisation.name } %>
 
 <% if SchemePolicy.new(current_user, nil).create? %>
-  <%= govuk_button_link_to "Create a new supported housing scheme", new_scheme_path, html: { method: :post } %>
+  <div class="govuk-button-group govuk-!-margin-bottom-6">
+    <%= govuk_button_link_to "Create a new supported housing scheme", new_scheme_path, html: { method: :post } %>
+  </div>
```

Comment:
Minor change to styling, you can see/not see the difference by clicking through the pages on the review app (as a support user) and not seeing a small difference between the position of the filters and lists on the sales log page and the same elements on the schemes page.